test/utils/ReceiptDataHelperSpec.scala

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package utils
+
+import java.time.LocalDate
+
+import base.BaseSpec
+import connectors.VatSubscriptionConnector
+import models.auth.User
+import models.errors.BadRequestError
+import models.nrs._
+import models.{CustomerDetails, SubmitVatReturnModel}
+import play.api.mvc.{AnyContentAsEmpty, Cookie}
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.play.views.helpers.MoneyPounds
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class ReceiptDataHelperSpec extends BaseSpec {
+
+  val mockConnector: VatSubscriptionConnector = mock[VatSubscriptionConnector]
+
+  val service = new ReceiptDataHelper(mockConnector, messagesApi)
+
+  val dateToUse: LocalDate = LocalDate.now()
+
+  def createReturnModel(flatRateScheme: Boolean): SubmitVatReturnModel = SubmitVatReturnModel(
+    10, 25.55, 33, 74, 95.06, 1006, 1006.66, 889.9, 900.0, flatRateScheme, dateToUse, dateToUse, dateToUse
+  )
+
+  def expectedAnswers(frs: Boolean, language: Language): Seq[Answers] = {
+
+    val box1Expected = if(language == EN) {
+      "VAT you charged on sales and other supplies"
+    } else {
+      "I`M WELSH INIT BRUH"
+    }
+
+    val boxSixExpected = (frs, language) match {
+      case (true, EN) => "Total value of sales and other supplies, including VAT"
+      case (false, EN) => "Total value of sales and other supplies, excluding VAT"
+      case (true, CY) => "Total value of sales and other supplies, including VAT"
+      case (false, CY) => "Total value of sales and other supplies, excluding VAT"
+    }
+
+    Seq(Answers(
+      "Nine box submission",
+      Seq(
+        ("box1", box1Expected, 10.00),
```

Comment:
Could you update the expected values to be in the format £10.00 and £1,006.00 etc instead of calling the MoneyPounds helper in the spec please? Need to make sure it does what we think it does instead of asserting the value against itself